testing/trino-tests/src/test/java/io/trino/server/testing/TestSystemSecurityMetadata.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.security;
+package io.trino.server.testing;
```

Comment:
`testing` is for testing-related utilities that are (unfortunately) part of production jar

a test (TestSystemSecurityMetadata) should not be in the testing package, please move it back